internal/setup/job.go

```diff
@@ -88,12 +81,14 @@ func NewJob(t *optimizev1beta2.Trial, mode string) (*batchv1.Job, error) {
 			Name:  fmt.Sprintf("%s-%s", job.Name, task.Name),
 			Image: task.Image,
 			Args:  task.Args,
-			Env: []corev1.EnvVar{
+			Env: append([]corev1.EnvVar{
 				{Name: "NAMESPACE", Value: t.Namespace},
 				{Name: "NAME", Value: task.Name},
 				{Name: "TRIAL", Value: t.Name},
 				{Name: "MODE", Value: mode},
 			},
+				task.Env...,
+			),
```

Comment:
I'm not 100% sure, but I think "last def wins", in which case I think we the explicit values at the end of the list.